scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableUnlessConfig.scala

```diff
@@ -0,0 +1,62 @@
+package scalafix.internal.config
+
+import metaconfig.{Conf, ConfDecoder, ConfError, Configured}
+import org.langmeta.Symbol
+
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+import scalafix.internal.util._
+
+case class UnlessConfig(
+    block: Symbol.Global,
+    symbol: Symbol.Global,
+    message: Option[String])
```

Comment:
FWIW, we recently added a `CustomMessage[T]` helper for exactly this pattern https://github.com/scalacenter/scalafix/blob/f5e0a1c084256e42cf6725208d5c13113f3f4821/scalafix-core/shared/src/main/scala/scalafix/config/CustomMessage.scala I supports multiline message descriptions and also the ability to customize the category id. However, it seems to only supports single field types which won't work here for `block` and `symbol`, I'll think how we can accommodate that :D